test/nim-tests.ts

```diff
@@ -26,9 +26,9 @@ import path from 'path';
 
 import {unwrap} from '../lib/assert';
 import {NimCompiler} from '../lib/compilers/nim';
+import {LanguageKey} from '../types/languages.interfaces';
```

Comment:
this seems unrelated: did the autoformatter do this? :( we are still trying to get the formatting and pre-commit hooks super reliable...

Reply:
Ah you said in your OG post, ignore, sorry!